src/test/common_files.spec.ts

```diff
@@ -0,0 +1,32 @@
+
+import { SecRunner, SecScan } from '@sec-tester/runner'
+import { TestType, Severity } from '@sec-tester/scan'
+import { Configuration } from "@sec-tester/core";
+
+describe('COMMON_FILES', () => {
+  let runner: SecRunner;
+  let scan: SecScan;
+   let configuration: Configuration = new Configuration({
+    hostname: process.env.BRIGHT_CLUSTER
+  });
+
+
+  beforeEach(async () => {
+    runner = new SecRunner(configuration);
+    await runner.init();
+
+  });
+
+  afterEach(async () => {
+    await runner.clear();
+  });
```

Comment:
Get rid of redundant async/await statements and brackets:
```suggestion
  afterEach(() => runner.clear());
```

Reply:
Wrote according to the documentation 😜:
https://github.com/NeuraLegion/sec-tester-js#:~:text=afterEach(async%20()%20%3D%3E%20%7B%0A%20%20%20%20await%20runner.clear()%3B%0A%20%20%7D)%3B